### PR TITLE
Update rust.md

### PR DESCRIPTION
**Pull request by danieltprice**
Initial Rust topic revisions.
https://websitemain62807-dpricerusttopicrevisions.gtsb.io/docs/integrations/rust/

### CODE REVIEW

File: content/docs/integrations/rust.md [hunk 1]
```diff
@@ -17,4 +36,11 @@ fn main() {
 }
 ```
 
-[On rust-lang playground](https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=0d9daa9cde3c74d2916c8f05b24707a3)
+where:
+
+- `<user>` is the database user, which is found on the project **Dashboard**, under **Connection Details**.
+- `<dbname>` is the database name (the default Neon project database is `main`).
+- `<password>` is the database user's password, which is provided to you when you create a Neon project.
+- `<project_id>` is the ID of the Neon project, which is found on the project **Settings** tab, under **General Settings**.
```
Comment (SomeoneToIgnore) [inline content/docs/integrations/rust.md:44]: That parameter seems to be unused in the code above?

Also `dbname` is `db_name` there.

Comment (danieltprice) [reply content/docs/integrations/rust.md]: Fixed. Thank you!

**Review by SomeoneToIgnore — COMMENTED** [review]
Great, now let's update the playground link accordingly (https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=0d9daa9cde3c74d2916c8f05b24707a3 now has different id names)

I wonder, whether we need such playground link at all, but have no real objections against it either.

File: content/docs/integrations/rust.md (hunk 1 above)
Comment (danieltprice) [reply content/docs/integrations/rust.md]: Dropped the rust playground link. The example is now out of sync with the doc.

**Review by SomeoneToIgnore — APPROVED** [review]

**Pull request merged**